.github/workflows/ci_tests.yaml

```diff
@@ -154,9 +153,21 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4.2.0
+
+      - name: Install dvc
+        run: |
+          uv venv
+          source .venv/bin/activate
+          uv pip install dvc
+          uv pip list
+
       # Pull baseline image data from dvc remote (DAGsHub)
       - name: Pull baseline image data from dvc remote
-        run: dvc pull --no-run-cache --verbose && ls -lhR pygmt/tests/baseline/
+        run: |
+          source .venv/bin/activate
+          uv run dvc pull --no-run-cache --verbose && ls -lhR pygmt/tests/baseline/
```

Comment:
Maybe we can combine the "Install dvc" and "Pull baseline image data from dvc remote" steps into one step "Setup DVC and pull baseline image data from dvc remote", so we don't have to `source .venv/bin/activate` twice.

Reply:
Let's just keep this as is, since I'm assuming this will be a temporary workaround until conda-forge's `dvc` supports Python 3.13. We might want to revert the changes here after a few months, so best to have two separate steps.